dhcp.go

```diff
@@ -56,6 +58,12 @@ func (d dhcpHandler) serveDHCP(w dhcp4.ReplyWriter, req *dhcp4.Packet) {
 		return
 	}
 
+	metrics.DHCPTotal.WithLabelValues(req.GetMessageType().String(), gi.String()).Inc()
+	labels := prometheus.Labels{"from": "dhcp", "op": req.GetMessageType().String()}
+	metrics.JobsTotal.With(labels).Inc()
+	metrics.JobsInProgress.With(labels).Inc()
```

Comment:
why not a `defer`?

Reply:
because since j.ServeDHCP is being handled in a go routine, i don't want it to mark the job as complete until that has been served. So I wait for it to complete first before cleaning up

Reply:
derp! makes sesnse!